<?xml version="1.0"?>
<launch>
    <node pkg="tl_detector" type="tl_detector_train.py" name="tl_detector_train" output="screen" cwd="node">
        <param name="traffic_light_is_close" value="30.0" />        
        <param name="ground_truth_dir" value="data_gt" />        
        <param name="train_data_dir" value="data_train" />        
        <param name="task" value="trainList1" />                
    </node>
</launch>
